osf/models/metaschema.py

```diff
@@ -10,12 +11,29 @@
 
 from website.project.metadata.utils import create_jsonschema_from_metaschema
 
+FORMBLOCK_TYPES = [
+    ('string', 'string'),
+    ('singleselect', 'singleselect'),
+    ('multiselect', 'multiselect'),
+    ('osf-author-import', 'osf-author-import'),
+    ('osf-upload', 'osf-upload'),
+    ('header', 'header'),
+]
+
+FORMBLOCK_SIZES = [
+    ('sm', 'sm'),
+    ('md', 'md'),
+    ('lg', 'lg'),
+    ('xl', 'xl'),
+]
+
 
 class AbstractSchema(ObjectIDMixin, BaseModel):
     name = models.CharField(max_length=255)
     schema = DateTimeAwareJSONField(default=dict)
     category = models.CharField(max_length=255, null=True, blank=True)
     active = models.BooleanField(default=True)
+    config = DateTimeAwareJSONField(blank=True, default=dict)
```

Comment:
If there's not an immediate use case for other schemas, can we just slap this on the registration schema for now?